free-courses-vi.md

```diff
@@ -172,6 +172,7 @@
 ### React
 
 * [React.js Cơ Bản](https://www.youtube.com/watch?v=zrdXI6uZuW4&list=PLzrVYRai0riSPcINVFvaCaM7Ul55DzpLd)
+* [Đồ án React cơ bản](https://www.youtube.com/watch?v=yYM1edV0j4I&list=PLv6GftO355Av08p6Zi1I67VYw47nMS8xO) - ZendVN
```

Comment:
We prefer YouTube URLs to be to the `/playlist` endpoint rather than the `/watch` endpoint. Could you update this so users can get a better overview of the entire course?

```suggestion
* [ReactJS - Demo các project trong khóa học](https://www.youtube.com/playlist?list=PLv6GftO355Av08p6Zi1I67VYw47nMS8xO) - ZendVN - Học Lập Trình Online
```